pdns/dynhandler.cc

```diff
@@ -269,8 +269,8 @@ string DLNotifyHandler(const vector<string>&parts, Utility::pid_t ppid)
   UeberBackend B;
   if(parts.size()!=2)
     return "syntax: notify domain";
-  if(!::arg().mustDo("master"))
-      return "PowerDNS not configured as master";
+  if(!::arg().mustDo("master") && !::arg().mustDo("slave-renotify"))
+      return "PowerDNS not configured as master or slave with re-notifications";
```

Comment:
I do not think this is useless. The code below is fine. Just the check was wrong: NOTIFYs can be sent either if PDNS has master mode active OR slave-renotify is enabled.
In my tests everthing worked fine.